examples/persistent_subscriptions.rs

```diff
@@ -92,3 +92,108 @@ async fn delete_persistent_subscription(client: &Client) -> eventstore::Result<(
 
     Ok(())
 }
+
+async fn get_persistent_subscription_to_stream_info(client: &Client) -> eventstore::Result<()> {
+    // #region get-persistent-subscription-to-stream-info
+    let info = client
+        .get_persistent_subscription_info("test-stream", "subscription-group", &Default::default())
+        .await?;
+
+    println!(
+        "GroupName: {}, EventStreamId: {}, Status: {:?}",
+        info.group_name, info.event_stream_id, info.status
+    );
+    // #endregion get-persistent-subscription-to-stream-info
+    Ok(())
+}
+
+async fn get_persistent_subscription_to_all_info(client: &Client) -> eventstore::Result<()> {
+    // #region get-persistent-subscription-to-all-info
+    let info = client
+        .get_persistent_subscription_info("$all", "subscription-group", &Default::default())
```

Comment:
**CONCERN:** This has test only for the regular stream subscription.

Reply:
Done